src/front/index.html

```diff
@@ -9,5 +9,11 @@
 </head>
 <body>
     <div id="root"></div>
+    <div id="deviceready"
```

Comment:
We should take care to remove it after tuning up cordova environment
It will be nice to place TODO comment here just to keep in mind